mpc_controller_node:
  ros__parameters:
    du_threshold: 0.001
    dt: 0.02
    max_iter: 150
    predict_steps: 30
    delay_num: 0
    max_omega: 24.0
    max_domega: 10.0
    max_speed: 0.8
    min_speed: -0.8
    max_accel: 10.0
    test_mpc: false
    bk_mode: false
    matrix_q: [100.0, 100.0, 2.0]
    matrix_r: [0.01, 0.01]
    matrix_rd: [0.01, 100.0]
    max_steer: 0.505
    max_dsteer: 1.5
    wheel_base: 0.26
    model_type: 2